lib/ansible/plugins/lookup/cyberarkpassword.py

```diff
@@ -144,9 +144,9 @@ def get(self):
                     if "passprops" not in result_dict:
                         result_dict["passprops"] = {}
                     output_prop_name = output_names[i][10:]
-                    result_dict["passprops"][output_prop_name] = output_values[i]
+                    result_dict["passprops"][output_prop_name] = to_native(output_values[i])
```

Comment:
`credential` from line 127 should be initialed as bytes, not a native string.
```python
                b_credential = b""
```

This makes the code on line 140 read more explicitly.
```python
            output_values = b_credential.split(b_delimiter)
```

Reply:
Miking bytes and strings in the arguments to `Popen()` is asking for trouble if there are non-ASCII characters in some of the arguments. It is best to convert all the arguments to bytes before passing to `Popen()`.

```python
b_all_params = [to_bytes(v) for v in all_params]
```

Reply:
Thanks a lot, I just pushed a new commit to this PR to address this comment.